modules/monitoring-modifying-core-platform-alerting-rules.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-core-platform-alerting-rules_{context}"]
+= Modifying core platform alerting rules
+
+As a cluster administrator, you can modify core platform alerting rules to suit your needs. For example, you can tweak an alert expression or change the severity label of an alert.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
```

Comment:
I think we should add
edit featuregate to enable TechPreview feature
`# oc edit featuregate/cluster`
then change the spec to:
```
spec:
  featureSet: TechPreviewNoUpgrade
```

Reply:
Added a step to enable TP features and added a new line to the sample per your feedback.